sdk/spring/spring-cloud-azure-stream-binder-eventhubs/README.md

```diff
@@ -40,6 +41,17 @@ that's why you have to fill `spring.cloud.stream.eventhub.checkpoint-storage-acc
 Event Hub provides a similar concept of physical partition as Kafka. But unlike Kafka's auto rebalancing between consumers and partitions, Event Hub provides a kind of preemptive mode. The storage account acts as a lease to determine which partition is owned by which consumer. When a new consumer starts, it will try to steal some partitions
 from most heavy-loaded consumers to achieve the workload balancing.
 
+#### Batch Consumer Support
+Azure Event Hubs Spring Cloud Stream Binder supports [Spring Cloud Stream Batch Consumer feature][spring-cloud-stream-batch0-consumer].
```

Comment:
[Spring Cloud Stream Batch Consumer feature][spring-cloud-stream-batch0-consumer] is not render correctly

Reply:
updated